telegram/ext/updater.py

```diff
@@ -149,15 +149,15 @@ def _thread_wrapper(self, target, *args, **kwargs):
             target(*args, **kwargs)
         except Exception:
             self.__exception_event.set()
-            self.logger.exception('unhandled exception')
+            self.logger.exception('unhandled exception in %s', thr_name)
             raise
         self.logger.debug('{0} - ended'.format(thr_name))
 
     def start_polling(self,
                       poll_interval=0.0,
                       timeout=10,
                       clean=False,
-                      bootstrap_retries=0,
+                      bootstrap_retries=-1,
```

Comment:
The docstring doesn;t reflect this change to the default value

Reply:
will fix